.flowconfig

```diff
@@ -1,3 +1,8 @@
+[options]
+suppress_comment=\\(.\\|\n\\)*\\$ExpectError
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe
+include_warnings=true
```

Comment:
I remember there was an issue with this in older versions of Flow, where it wasn't failing the flow check if there were warnings, like `$ExpectError` being unused (what does interest us). But I don't know if our version of Flow has this problem. Could you check this?

Reply:
Nice! Sad that it's not available in the flow config.